python/src/opendp/measurements.py

```diff
@@ -1139,7 +1139,7 @@ def make_report_noisy_max_gumbel(
     :raises UnknownTypeException: if a type argument fails to parse
     :raises OpenDPException: packaged error from the core OpenDP library
     """
-    assert_features("contrib")
+    assert_features("contrib", "floating-point")
```

Comment:
Not the usual direction. Maybe it got removed accidentally earlier?

Reply:
I removed the earlier diff so as to be in its own PR.